xm-commons-lep/src/main/java/com/icthh/xm/commons/lep/XmLepResourceService.java

```diff
@@ -219,7 +220,12 @@ private String getTenantScriptLocation(String path, ContextsHolder contextsHolde
 
             case FILE: {
                 String lepDir = Paths.get(FileSystemUtils.APP_HOME_DIR, "config", "tenants",
-                                          tenantKey.toUpperCase(), appName, "lep").toString();
+                    tenantKey.toUpperCase(), appName, "lep").toString();
+
+                if (SystemUtils.IS_OS_WINDOWS) {
+                    return "file:///" + lepDir + path;
```

Comment:
user methods from lib

Reply:
Please concretize about what methods from lib you are talking about and where exactly they should be used.

Reply:
any, for example
https://commons.apache.org/proper/commons-io/javadocs/api-1.4/org/apache/commons/io/FilenameUtils.html

Reply:
Remark applied.

Reply:
remark applied.